storage/migrations/14_events_related_accounts.up.sql

```diff
@@ -0,0 +1,155 @@
+BEGIN;
+
+CREATE TABLE chain.events_new
+(
+    tx_block UINT63 NOT NULL,
+    tx_index  UINT31,
+    FOREIGN KEY (tx_block, tx_index) REFERENCES chain.transactions(block, tx_index) DEFERRABLE INITIALLY DEFERRED,
+
+    event_index UINT31 NOT NULL, -- Added.
+    type    TEXT NOT NULL,  -- Enum with many values, see ConsensusEventType in api/spec/v1.yaml.
+    PRIMARY KEY (tx_block, type, event_index), -- Added.
+
+    body    JSONB,
+    tx_hash   HEX64, -- could be fetched from `transactions` table; denormalized for efficiency
+    -- related_accounts TEXT[], -- Removed.
+
+    -- There's some mismatch between oasis-core's style in Go and nexus's
+    -- style in SQL and JSON. oasis-core likes structures filled with nilable
+    -- pointers, where one pointer is non-nil. nexus likes a type string plus
+    -- a "body" blob. In roothash events though, the roothash/api Event
+    -- structure additionally has a RuntimeID field, which nexus otherwise
+    -- loses when it extracts the one non-nil event next to it. In the nexus,
+    -- database, we're storing that runtime identifier in a column.
+    --
+    -- This is a runtime identifier, which is a binary "namespace," e.g.
+    -- `000000000000000000000000000000000000000000000000f80306c9858e7279` for
+    -- Sapphire on Mainnet. This is taken from the event from the node, and it
+    -- is set even for runtimes that nexus isn't configured to analyze.
+    roothash_runtime_id HEX64,
+    roothash_runtime runtime,
+    roothash_runtime_round UINT63
+);
+
+CREATE TABLE chain.events_related_accounts
```

Comment:
not replicated as a comment in 00_consensus.up.sql. should we? I noticed you did for the runtimes schema

Reply:
Updated, thanks!